smart-campus-service-api/src/main/java/hu/unideb/smartcampus/service/ejabberd/SharedRosterService.java

```diff
@@ -0,0 +1,79 @@
+package hu.unideb.smartcampus.service.ejabberd;
+
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Service for managing shared roster service by Java side.
+ *
+ */
+public interface SharedRosterService {
+
+  /**
+   * Endpoint URL for list all shared roster group.
+   */
+  String SHARED_ROSTER_GROUP_LIST_COMMAND = "/srg_list";
```

Comment:
Instead of this, what do you think a simple java class which has these fields?

I think that is not a good way to define public static final variables in interfaces.
(Maybe you can check similar classes in our shared-module, but this new class can stay in this module if you want.)